util/parsenum.h

```diff
@@ -121,8 +121,8 @@ _Pragma("clang diagnostic pop")
 			    (min) <= 0 ? 0 : (min),			\
 			    (uintmax_t)(max), *(x), (base),		\
 			    (trailing))),				\
-			((((max) < 0) && (errno == 0)) ?		\
-			    (errno = ERANGE) : 0)),			\
+				((((max) < 0) && (errno == 0)) ?	\
```

Comment:
Based on the parentheses, these two lines belong to the `parsenum_unsigned()` branch, so I think they should be indented.  But maybe you thought it was easier to read in the original format?  This indentation was present in the very first parsenum commit: https://github.com/Tarsnap/libcperciva/commit/0461d4159cb701700568e937402f104d5a309fd9#diff-6c2e648c26c72487e2702c8daeea8629R30